experiments/AtmosLES/dycoms.jl

```diff
@@ -376,9 +375,9 @@ function main()
     timeend = FT(100)
 
     driver_config = config_dycoms(FT, N, resolution, xmax, ymax, zmax)
-    solver_config = CLIMA.setup_solver(t0, timeend, driver_config, init_on_cpu=true)
+    solver_config = CLIMA.setup_solver(t0, timeend, driver_config, Courant_number=1.8, init_on_cpu=true)
```

Comment:
```suggestion
    solver_config = CLIMA.setup_solver(t0, timeend, driver_config; Courant_number=1.8, init_on_cpu=true)
```
Assuming all kwargs come after `driver_config`

Reply:
Fixed :D